sources/web/datalab/polymer/index.dark.css

```diff
@@ -12,16 +12,13 @@
  * the License.
  */
 
-body {
-  margin: 0px;
-  padding: 0px;
-  height: 100%;
-}
-datalab-app {
-  position: absolute;
-  top: 0;
-  bottom: 0;
-  left: 0;
-  right: 0;
-  min-width: 470px;
+:root {
+  --primary-accent-color: #222;
+  --primary-bg-color: #222;
+  --secondary-bg-color: #2b2b2b;
+  --primary-fg-color: #aaa;
+  --selection-bg-color: #444;
+  --selection-fg-color: #d68600;
+  --disabled-bg-color: var(--primary-bg-color);
```

Comment:
It's just that it's a different state, so I think it's good to separate its variable out, even if it has the same value.
I'm ok removing this if you still don't think it should be there.